manifests/templates/reconciler-manager-configmap.yaml

```diff
@@ -181,18 +181,14 @@ data:
                cpu: "50m"
                memory: "200Mi"
          - name: otel-agent
-           image: gcr.io/config-management-release/otelcontribcol:v0.54.0-gke.1
+           image: gcr.io/config-management-release/otelcontribcol:v0.86.0-gke.1
            command:
            - /otelcontribcol
            args:
            - "--config=/conf/otel-agent-config.yaml"
-           resources:
-             limits:
-               cpu: 1
-               memory: 1Gi
-             requests:
-               cpu: 10m
```

Comment:
Why is this removed? It is not removed in #916 that you're cherrypicking from.

Reply:
looks like some merge conflict wasn't resolved correctly in the previous commit, should be fixed now